js/interface.js

```diff
@@ -1,3 +1,7 @@
+// To run experiment
+// To be removed once experiment is over
+Fliplet.TagManager.init();
```

Comment:
If there's a possibility that we'll be running experiments on widget interface and providers, do we maybe want to run `Fliplet.TagManager.init()` globally? I'm aware this would mean changing the core JS/template, but it could help a lot. It also touches on the comment "To be removed once experiment is over" -- why does it need to be removed?

@squallstar Opinion?

Reply:
There's definitely a possibility of us running experiments on the widget interface as well as providers like this one.
I wondered the same, but I also kinda get why this was implemented this way, because not all components need to have GTM running all the time, I can't even say how often we will be running experiments on widgets.
In any case I would like to hear from @squallstar .

Reply:
@hcarneiro @tonytlwu if we ever want to do this globally I suggest adding a new property in `widget.json` (e.g. `interface_require_gtm: true` which is then used by the engine to output a flag in the frontend and running `Fliplet.TagManager.init();`.

Do you agree?

@hcarneiro how many components do require GTM in the next 2 months? If it's less than say 4 or 5 then I don't think we should be adding a core feature.

Reply:
@squallstar currently we have plans for 3 - Analytics, Apple AAB, and Google AAB.

Reply:
I can see a lot of room for more, if we consider the current Growth plans. If they needed to experiment with:

- Making Fliplet's app building process focussed on data-first: data source manager, data source provider etc.
- Promoting push notifications and how to use them: push notifications provider
- Testing how file capacity limitation and warnings are presented to users: file manager, file picker
- Providing recommended guidance and wizard for setting up theme appearance to match branding: theme manager
- etc.

It could expand quite easily.

Reply:
@tonytlwu that is true. I forgot about the data source provider and notifications. We definitely will touch on those.

So maybe this should be done globally.